`rm` command succeeds even when bookmark does not exist

## Description
The `delete` command currently reports success even if the specified bookmark alias does not exist in the database.

Example:

```sh
lynx rm git
```

When `git` is not present in the database, the command completes silently because the underlying SQL `DELETE` affects 0 rows. This gives the impression that the operation succeeded.

## Expected behevior

* If no bookmark matches the given alias, the command should return an error or warning (e.g., `[ERROR] no bookmark with alias "git"`).

--- a/lynx.c
+++ b/lynx.c
@@ -140,6 +140,9 @@ void remove_bookmark(sqlite3 *db, char *alias) {
   if (sqlite3_step(stmt) != SQLITE_DONE) {
     nob_log(ERROR, "%s", sqlite3_errmsg(db));
   }
+  if (sqlite3_changes(db) == 0) {
+    nob_log(ERROR, "no bookmark found with alias %s", alias);
+  }
 }
 
 int main(int argc, char *argv[]) {

